src/libstd/net/mod.rs

```diff
@@ -15,8 +15,25 @@
 //!   with networking objects like [`TcpListener`], [`TcpStream`] or [`UdpSocket`]
 //! * Other types are return or parameter types for various methods in this module
 //!
+//! # IP addresses helpers stability guarantees
+//!
+//! [`IpAddr`], [`Ipv4Addr`] and [`Ipv6Addr`] offer helper methods
+//! that provide information about an address, for instance whether it
+//! is globally routable (see [`IpAddr::is_global()`]), or if it is a
+//! multicast address (see [`IpAddr::is_multicast()`]). These methods
+//! are compliant with the [IETF RFCs]. As erratas and new RFCs are
+//! published, these methods behavior may be subject to changes. For
+//! instance, if the `240/4` IPv4 blocks which is currently reserved
+//! for "future use" is made globally routable by an RFC,
+//! [`Ipv4Addr::is_reserved()`] would return `false` for such
```

Comment:
not sure if this should be "would return" or "will return".

Reply:
`will` sounds better